searcher/resource/resource.go

```diff
@@ -0,0 +1,23 @@
+package resource
+
+type Resource struct {
+	Document *Document `json:"document"`
+	Meta     *Meta     `json:"meta"`
+}
+
+// type raw struct {
+// 	meta meta `json:"meta"`
+// 	document []byte `json:"document"`
+// }
+
+type Meta struct {
+	HTTPStatus int    `json:"http_status"`
+	URL        string `json:"url"`
+}
+
+type Document struct {
+	Title     string   `json:"title"`
+	Content   []string `json:"content"`
```

Comment:
Is the content a set of sentences or what is the slicing logic behind this list?

Reply:
As we perform multiple steps of cleaning for the content, would it be interesting to store that cleaned content within the struct maybe?

Reply:
> Is the content a set of sentences or what is the slicing logic behind this list?

It is a slice of sentences (taken from `<p>` tags in the scraper)

Reply:
> As we perform multiple steps of cleaning for the content, would it be interesting to store that cleaned content within the struct maybe?

Potentially, my thinking was that it only gets analysed and the topics extracted from it once so there would be no need to additionally store it. However I guess if it was necessarily to use it later it would be worth having it stored.